src/app/data-planes/sources.ts

```diff
@@ -103,7 +107,7 @@ export const sources = (api: KumaApi) => {
     },
 
     '/meshes/:mesh/dataplane-overviews/:name': async (params) => {
-      return api.getDataplaneOverviewFromMesh(params)
+      return DataplaneOverview.fromObject(await api.getDataplaneOverviewFromMesh(params), can('use zones'))
```

Comment:
This is more of a self-note/let you know that I'm still looking/thinking about this. Ideally `zones` should be dealt with in zones. I need to refresh my mind on this to decide what (if anything) to do here though.